Source/AliveLibAE/Abe.cpp

```diff
@@ -57,13 +57,10 @@
 using TAbeStateFunction = decltype(&Abe::State_0_Idle_44EEB0);
 
 #define MAKE_STRINGS(VAR) #VAR,
-const s8* const sAbeStateNames[] =
-{
-    ABE_STATES_ENUM(MAKE_STRINGS)
-};
+const s8* const sAbeStateNames[] = {
```

Comment:
same for this

Reply:
I don't think there's an option specifically for this one, unfortunately